src/web_app/html/index_template.html

```diff
@@ -124,24 +124,23 @@
 {% if not chromeapp %}
   <script type="text/javascript">
     var loadingParams = {
```

Comment:
I originally did this but I wanted the granularity and be able to chose what is JSON and what's not due to some is already JSON. But yes, this is a much cleaner approach but it either requires cleaning up so that we only encode it once or we have to decode it twice.

I will redo this.